index.html

```diff
@@ -0,0 +1,553 @@
+              <a
+                href="https://twitter.com/irantsane"
+                class="about__icons__link link"
+                ><img src="images/twitter.png" alt="twitter icon"
+              /></a>
+            </li>
+            <li class="about__icons__li">
+              <a href="#" class="about__icons__link link"
+                ><img src="images/M.png" alt="M icon"
+              /></a>
+            </li>
+          </ul>
+          <button
+            type="button"
+            class="btn fw-500 fz-17 lh-24 clr-p-500 bg-white"
+          >
+            Get my resume
+          </button>
+        </div>
+        <ul class="about__tools ul">
+          <li class="about__lang">
+            <div class="about__lang__top flex">
+              <h3 class="about__lang__top__title fw-500 fz-20 lh-24 clr-n-900">
+                Languages
+              </h3>
+              <img
+                class="about__lang__top__img"
+                src="images/arrow.png"
+                alt="arrow icon"
+              />
+            </div>
+            <ul class="about__lang__bottom ul">
+              <li class="flex bg-n-300">
+                <img src="images/js.png" alt="javascript icon" />
+                <p
+                  class="about__lang__bottom__name fw-500 fz-15 lh-20 clr-n-700"
+                >
+                  JavaScript
+                </p>
+              </li>
+              <li class="flex bg-n-300">
+                <img src="images/html.png" alt="html icon" />
+                <p
+                  class="about__lang__bottom__name fw-500 fz-15 lh-20 clr-n-700"
+                >
+                  HTML
+                </p>
+              </li>
+              <li class="flex bg-n-300">
+                <img src="images/css.png" alt="css icon" />
+                <p
+                  class="about__lang__bottom__name fw-500 fz-15 lh-20 clr-n-700"
+                >
+                  CSS
+                </p>
+              </li>
+            </ul>
+          </li>
+          <li class="about__frameworks">
+            <div class="about__frameworks__top flex">
+              <h3
+                class="about__frameworks__top__title fw-500 fz-20 lh-24 clr-n-900"
+              >
+                Frameworks
+              </h3>
+              <img
+                class="about__frameworks__top__img"
+                src="images/arrow.png"
+                alt="arrow icon"
+              />
+            </div>
+          </li>
+          <li class="about__skills">
+            <div class="about__skills__top flex">
+              <h3
+                class="about__skills__top__title fw-500 fz-20 lh-24 clr-n-900"
+              >
+                Skills
+              </h3>
+              <img
+                class="about__skills__top__img"
+                src="images/arrow.png"
+                alt="arrow icon"
+              />
+            </div>
+          </li>
+        </ul>
+      </section>
```

Comment:
- [ ] As requested of you by the previous reviewer, kindly reduce the margins/ paddings left and right of this section so that it may match the provide Figma design.

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/201472605-95589231-3e79-4239-988d-dadf36a88310.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/201472636-f4ce1033-14c5-4c72-84c7-cec4e6424047.png)